travis-bootstrap.sh

```diff
@@ -42,11 +42,11 @@ timed cabal update
 # issue, and it is easy enough to work around: clean first.
 
 echo Cabal
-(cd Cabal && timed cabal clean)
```

Comment:
Alternatively we could use `bash` everywhere and `{ }` syntax, i.e. no subshells. But ah.

I was thinking of writing own shell scripting language just to make CI stuff easier, but the pain and frustration levels haven't risen enough yet.